src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/InnerApi/Requests/GetVacanciesRequest.cs

```diff
@@ -4,46 +4,26 @@
 
 namespace SFA.DAS.FindAnApprenticeship.InnerApi.Requests
 {
-    public class GetVacanciesRequest : IGetApiRequest
+    public class GetVacanciesRequest(
+        double? lat,
+        double? lon,
+        int? distance,
+        string? searchTerm,
+        int? pageNumber,
+        int? pageSize,
+        IReadOnlyCollection<string> categories,
+        IReadOnlyCollection<string> levels,
+        VacancySort sort,
+        WageType? skipWageType,
+        bool disabilityConfident)
+        : IGetApiRequest
     {
-        private readonly double? _lat;
-        private readonly double? _lon;
-        private readonly int? _distance;
-        private readonly int? _pageNumber;
-        private readonly int? _pageSize;
-        private readonly string _categories;
-        private readonly string _levels;
-        private readonly string _searchTerm;
-        private readonly VacancySort _sort;
-        private readonly bool _disabilityConfident; 
-
-        public GetVacanciesRequest(
-            double? lat,
-            double? lon,
-            int? distance,
-            string? searchTerm,
-            int? pageNumber,
-            int? pageSize,
-            IReadOnlyCollection<string> categories,
-            IReadOnlyCollection<string> levels,
-            VacancySort sort,
-            bool disabilityConfident)
-        {
-            _lat = lat;
-            _lon = lon;
-            _distance = distance;
-            _sort = sort;
-            _pageNumber = pageNumber;
-            _pageSize = pageSize;
-            _categories = categories is { Count: > 0 } ? string.Join("&categories=", categories) : string.Empty;
-            _levels = levels is { Count: > 0 } ? string.Join("&levels=", levels) : string.Empty;
-            _searchTerm = searchTerm;
-            _disabilityConfident = disabilityConfident;
-
-        }
+        private readonly string _categories = categories is { Count: > 0 } ? string.Join("&categories=", categories) : string.Empty;
+        private readonly string _levels = levels is { Count: > 0 } ? string.Join("&levels=", levels) : string.Empty;
 
 
         public string Version => "2.0";
-        public string GetUrl => $"/api/vacancies?lat={_lat}&lon={_lon}&distanceInMiles={_distance}&sort={_sort}&pageNumber={_pageNumber}&pageSize={_pageSize}&categories={_categories}&levels={_levels}&searchTerm={_searchTerm}&disabilityConfident={_disabilityConfident}";
+        public string GetUrl =>
```

Comment:
Perfect example of potential problems with empty types. You might find on the other end of this that categories gets created as an empty list with a single null item in it, if it is passed as an empty string.